bindings/java/src/lib.rs

```diff
@@ -15,71 +15,49 @@
 // specific language governing permissions and limitations
 // under the License.
 
-use std::cell::RefCell;
 use std::collections::HashMap;
 use std::ffi::c_void;
 use std::str::FromStr;
-use std::sync::Arc;
 
 use jni::objects::JClass;
 use jni::objects::JMap;
 use jni::objects::JObject;
 use jni::objects::JString;
 use jni::objects::JThrowable;
 use jni::objects::JValue;
-use jni::sys::jboolean;
-use jni::sys::jint;
-use jni::sys::jlong;
-use jni::sys::JNI_VERSION_1_8;
-use jni::JNIEnv;
-use jni::JavaVM;
+use jni::sys::{jboolean, jobject, JNI_VERSION_1_8};
+use jni::sys::{jint, jlong};
+use jni::{JNIEnv, JavaVM};
 use once_cell::sync::OnceCell;
+use tokio::runtime::Builder;
+use tokio::runtime::Runtime;
+
 use opendal::BlockingOperator;
 use opendal::Operator;
 use opendal::Scheme;
-use tokio::runtime::Builder;
-use tokio::runtime::Runtime;
 
 static mut RUNTIME: OnceCell<Runtime> = OnceCell::new();
-
-thread_local! {
```

Comment:
We don't need thread-locals. Because:

1. JavaVM is globally the same one.
2. ENV can be retrieved locally while attaching the current thread as a daemon is idempotent so that we don't increase cost.

Reply:
The function "attach_current_thread_as_daemon" incurs overhead because it calls GetEnv via JNI. Additionally, calling this function in every asynchronous function seems strange. To improve clarity, why not cache the ENV in thread-local?

Reply:
> it calls GetEnv via JNI

Thanks for pointing it out. I'll revert to the thread local solution for ENV, while VM can be still global and detach happen implicitly on thread stop.

Reply:
Resolved by c8a3be0c2b77cac9ef682fc86bbc88017448fe03